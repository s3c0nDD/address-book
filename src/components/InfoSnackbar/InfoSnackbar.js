import React from 'react';

import Alert from '@material-ui/lab/Alert';
import Snackbar from '@material-ui/core/Snackbar';

const InfoSnackbar = ({ autoHideDuration, message, open }) => {
  return (
    <Snackbar
      autoHideDuration={autoHideDuration}
      open={open}
    >
      <Alert
        variant="filled"
        severity="info"
      >
        {message}
      </Alert>
    </Snackbar>
  )
};

export default InfoSnackbar;
