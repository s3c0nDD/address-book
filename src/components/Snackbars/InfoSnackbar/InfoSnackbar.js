import React from 'react';

import CommonSnackbar from '../CommonSnackbar';

const InfoSnackbar = ({ message, open }) => {
  return (
    <CommonSnackbar
      open={open}
      message={message}
      severity="info"
    />
  )
};

export default InfoSnackbar;
