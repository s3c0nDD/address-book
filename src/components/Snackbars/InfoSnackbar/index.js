import InfoSnackbar from './InfoSnackbar'

export default InfoSnackbar;