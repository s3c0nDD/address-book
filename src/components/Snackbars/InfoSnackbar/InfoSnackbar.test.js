import React from 'react';
import { createShallow } from '@material-ui/core/test-utils';

import InfoSnackbar from './InfoSnackbar';

describe('InfoSnackbar', () => {
  let shallow;
  let component;
  let wrapper;

  beforeAll(() => {
    shallow = createShallow();
  });

  beforeEach(() => {
    component = (
      <InfoSnackbar
        message="Hello world"
        open={true}
      />
    );
    wrapper = shallow(component);
  });

  it('shallowly renders and matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
