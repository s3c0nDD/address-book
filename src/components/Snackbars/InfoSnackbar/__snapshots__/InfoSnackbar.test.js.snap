// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoSnackbar shallowly renders and matches snapshot 1`] = `
<CommonSnackbar
  message="Hello world"
  open={true}
  severity="info"
/>
`;
